### PR TITLE
Fix/tests

### PR DESCRIPTION
**Pull request by alch**
* Test were fixed in all modules
* Fixed behaviors of managers that were not expecting paymentbridge amounts in *cents*
* removed `split.sh`
* added `.travis.yml`

### CODE REVIEW

**Comment by alch** [comment]
Ping @PaymentSuite/owners

File: src/PaymentSuite/RedsysBundle/Tests/Functional/Controller/RedsysControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * Test execute controller
+     */
+    public function testExecute()
+    {
+        $crawler = $this
+            ->client
+            ->request('GET', '/payment/redsys/execute');
+
+        //$this->assertTrue($client->getResponse()->isRedirect());
```
Comment (mickaelandrieu) [inline src/PaymentSuite/RedsysBundle/Tests/Functional/Controller/RedsysControllerTest.php]: can you remove this commented code please ?

File: src/PaymentSuite/RedsysBundle/Tests/Functional/Services/RedsysFormTypeWrapperTest.php [hunk 2]
```diff
@@ -155,11 +159,11 @@ public function setUp()
...
     public function testFormCreation()
     {
-        $amount = 10;
+        $amount = 100;
 
         $formData = array(
-            'Ds_Merchant_Amount'             => $amount * 100,
-            'Ds_Merchant_MerchantSignature'  => '11473E3E726C16798532E9924444953A5C12DB2C',
+            'Ds_Merchant_Amount'             => $amount,
+            'Ds_Merchant_MerchantSignature'  => 'CB43B12351A9826D9640CC285CBDFD8CA6A5994C',
```
Comment (mickaelandrieu) [inline src/PaymentSuite/RedsysBundle/Tests/Functional/Services/RedsysFormTypeWrapperTest.php]: can you explain me what is this identifier ? it is from Redsys platform ? do we need a real identifier ?

File: src/PaymentSuite/RedsysBundle/phpunit.xml.dist [hunk 3]
```diff
@@ -27,8 +27,5 @@
             </exclude>
         </whitelist>
     </filter>
-    <php>
```
Comment (mickaelandrieu) [inline src/PaymentSuite/RedsysBundle/phpunit.xml.dist]: Are you sure ? it's a recommandation from Symfony documentation => http://symfony.com/doc/current/book/testing.html#your-first-functional-test

**Comment by mickaelandrieu** [comment]
A lot of changes in only one pull request.

I think we need to create a special major release for v2 of stripe to let the stripe v1 users with the "old but functionnal" version: what do you think @alch  ?

File: src/PaymentSuite/RedsysBundle/Tests/Functional/Services/RedsysFormTypeWrapperTest.php (hunk 2 above)
Comment (xphere) [inline src/PaymentSuite/RedsysBundle/Tests/Functional/Services/RedsysFormTypeWrapperTest.php]: Yes, it is is Redsys specific. Signature is just a digest of the formData plus merchant information, like security keys, just for checking that the command is not altered with a man-in-the-middle attack or something similar.

**Comment by alch** [comment]
@mickaelandrieu correct, The PR was meant to be only to fix tests. The **stripe v2** change was incorrectly pushed. I will revert it and create a different PR

File: src/PaymentSuite/RedsysBundle/phpunit.xml.dist (hunk 3 above)
Comment (alch) [inline src/PaymentSuite/RedsysBundle/phpunit.xml.dist]: @mickaelandrieu I added a dependency for https://github.com/elcodi/TestCommonBundle, which allows tests extending the `Elcodi\Bundle\TestCommonBundle\Functional\WebTestCase` to auto-discover the `KERNEL_DIR`.
Setting the `KERNEL_DIR` explicitly in the `phpunit.xml.dist` will force you to have a single symfony application, while we have several packages (one for each payment bundle) that can have their own `AppKernel` class.

File: composer.json [hunk 4]
```diff
@@ -16,14 +16,27 @@
...
+    },
+    "require-dev": {
+        "phpunit/phpunit": "4.6.*",
+        "symfony/twig-bundle": ">=2.2.3",
+        "symfony/monolog-bundle": ">=2.2.3",
+        "symfony/browser-kit": ">=2.2.3",
+        "symfony/css-selector": ">=2.2.3",
+        "symfony/twig-bundle": ">=2.2.3",
+        "elcodi/core-bundle": "0.5.*",
+        "elcodi/test-common-bundle": "0.5.*"
```
Comment (mickaelandrieu) [inline composer.json]: @alch OK ! I missed that :) :+1:

**Comment by alch** [comment]
@mickaelandrieu :+1:

**Comment by mickaelandrieu** [comment]
thank you @alch :)

**Pull request merged**